niflow_manager/data/templates/base/spec.yml

```diff
@@ -0,0 +1,50 @@
+#  # requirement for the environment needed to run the analyses
+#requirements:
+#  # example below, see full Neurodocker spcification: https://github.com/ReproNim/neurodocker#supported-software
+#  base:
+#    image: debian:stretch
+#    pkg-manager: apt
+#  fsl:
+#    version: 5.0.10
+#  afni:
+#    version: latest
+#  miniconda:
+#    conda_install: [python=3.7]
+#
+#  # test specification
+#nfm_test:
```

Comment:
is there a reason this is called `nfm_test`. this is all `nfm`  anyway.

Reply:
didn't want to have "tests" as a key in "test"...

Reply:
i think that is ok to have `tests` with `test`.  `nfm_test` seems weird within the `nfm` context. you can use `test`